docs/example-scenario/infrastructure/image-modeling.md

```diff
@@ -0,0 +1,115 @@
+- Use a storage [replication option](/azure/storage/common/storage-redundancy) that that meets your availability and performance requirements. In this example, Avere vFXT is configured for high availability by default, with locally redundant storage (LRS). For load balancing, all VMs in this setup use the round-robin pattern to access vFXT exports.
+- If the backend storage will be consumed by both Windows clients and Linux clients, use Samba servers to support the Windows nodes. A [version](https://github.com/paulomarquesc/beegfs-template) of this example scenario based on BeeGFS uses Samba to support the scheduler node of the HPC workload (PhotoScan) running on Windows. A load balancer is deployed to act like a smart replacement for DNS round robin.
+- Run HPC applications using the VM type best suited for your [Windows](/azure/virtual-machines/windows/sizes-hpc) or [Linux](/azure/virtual-machines/linux/sizes?toc=%2fazure%2fvirtual-machines%2flinux%2ftoc.json) workload.
+- To isolate the HPC workload from the storage resources, deploy each in its own virtual network, then use virtual network [peering](/azure/virtual-network/virtual-network-peering-overview) to connect the two. Peering creates a low-latency, high-bandwidth connection between resources in different virtual networks and routes traffic through the Microsoft backbone infrastructure through private IP addresses only.
+
+### Security
+
+This example focuses on deploying a high-performance storage solution for an HPC workload and is not a security solution. Make sure to involve your security team for any changes.
+
+For added security, this example infrastructure enables all the Windows VMs to be domain-joined and uses Active Directory for central authentication. It also provides custom DNS services for all VMs. To help protect the environment, this template relies on [network security groups (NSGs)](/azure/virtual-network/security-overview). NSGs offer basic traffic filters and security rules.
+
+Consider the following options to further improve security in this scenario:
+
+- Use network virtual appliances such as Fortinet, Checkpoint, and Juniper.
+- Apply [role-based access control](/azure/role-based-access-control/overview) to the resource groups.
+- Enable VM [JIT](/azure/security-center/security-center-just-in-time) access if jump boxes are accessed via the Internet.
+- Use [Azure Key Vault](/azure/key-vault/quick-create-portal) to store the passwords used by administrator accounts.
+
+## Pricing
+
+The cost of running this scenario can vary greatly depending on on multiple factors.  Choices such as using [GPU-optimized](/azure/virtual-machines/linux/sizes-gpu) VMs (like this example), how much storage is required, and the amount of time to complete a job will determine your cost.
```

Comment:
The second sentence is a bit confusing. It's enumerating things that must be chosen, but the second clause is a requirement. I'd recast it to make clearer that the choice of storage type or allocation affects cost.

Reply:
Agreed, cleaned it up slightly to provide clarity.